content/about/_index.md

```diff
@@ -83,6 +83,9 @@ actionText = "Our Team"
 url = "/about/board/"
 actionText = "Board of Directors"
 
+[[blocks.cta]]
+url = "/about/impact/"
```

Comment:
The URL is /impact/, but maybe it should be /about/impact/. Either way, probably it makes more sense to have the actual file be content/about/impact.md just so it's with the other marketing pages.